### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.cognitiveservices.vision.face.models.error_py3.Error
  fullName: azure.cognitiveservices.vision.face.models.error_py3.Error
  inheritance:
  - inheritance:
    - type: builtins.object
    type: msrest.serialization.Model
  langs:
  - python
  module: azure.cognitiveservices.vision.face.models.error_py3
  name: Error
  source:
    id: Error
    path: azure-cognitiveservices-vision-face\azure\cognitiveservices\vision\face\models\error_py3.py
    remote:
      branch: master
      path: azure-cognitiveservices-vision-face\azure\cognitiveservices\vision\face\models\error_py3.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 15
  summary: 'Error body.

    '
  syntax:
    content: 'Error(*, code: str = None, message: str = None, **kwargs) -> None'
    parameters:
    - description: ''
      id: code
      type:
      - str
    - description: ''
      id: message
      type:
      - str
  type: class
  uid: azure.cognitiveservices.vision.face.models.error_py3.Error
references: []
